lib/assist/assist.go

```diff
@@ -58,6 +58,8 @@ const (
 	MessageKindAssistantPartialFinalize MessageType = "CHAT_PARTIAL_MESSAGE_ASSISTANT_FINALIZE"
 	// MessageKindSystemMessage is the type of Assist message that contains the system message.
 	MessageKindSystemMessage MessageType = "CHAT_MESSAGE_SYSTEM"
+	// MessageKindUIMessage is the type of Assist message that is presented to user as information, but not stored persistently in the conversation. This can include backend error messages and the like.
+	MessageKindUIMessage MessageType = "CHAT_MESSAGE_UI"
```

Comment:
Will this ever be anything other than an error?

Reply:
Honestly dunno, for now this rate-limit is the only case 🤷

Reply:
@justinas @ryanclark Can we just change the name to `CHAT_MESSAGE_ERROR` and keep using that type for all assist-related messages?

Reply:
Yeah I think we should